apps/condo/lang/ru.json

```
@@ -310,12 +310,16 @@
   "pages.condo.ticket.TicketChanges.notice.DeletedCreatedAt.description": "Упс, кажется пользователь, который внёс эти изменения, был удалён из системы",
   "pages.condo.ticket.TicketChanges.fetchMore": "Показать ещё {count}",
   "pages.index.PageTitle": "Аналитика",
-  "pages.organizations.CreateOrganizationButtonLabel": "Создать новую организацию",
-  "pages.organizations.CreateOrganizationModalTitle": "Форма создания новой организации",
+  "pages.organizations.CreateOrganizationMessage": "Давайте создадим вашу управляющую компанию",
+  "pages.organizations.CreateOrganizationButtonLabel": "Добавить организацию",
+  "pages.organizations.CreateOrganizationModalTitle": "Создание организации",
   "pages.organizations.EditOrganizationModalTitle": "Форма редактирования организации",
   "pages.organizations.OrganizationName": "Название",
+  "pages.organizations.Inn": "ИНН",
```

I know, that we will be confused, but the correct way to name the key is `pages.organizations.itn`, because in English "ИНН" is "Individual Tax-payer Number" (ITN).

```suggestion
  "pages.organizations.ITN": "ИНН",
```

---

it will be confusing as we have inn field in organization meta -> form with inn field 
So, using tin in translations will make chaos